README.md

```diff
@@ -42,3 +42,10 @@ activate your new improver environment
 ```
 conda activate improver
 ```
+
+## Pre-commit Hook
+OPTIONAL: A pre-commit hook can be added to facilitate the development of this code base.
+Ensure that you have python available on the path, then install the pre-commit hook by running `pre-commit install` from within your working copy.
+pre-commit checks will run against modified files when you commit from then on.
+
+These pre-commit hooks will run as part of continuous integration to maintain standards in the project.
```

Comment:
Which standards? Should this say "code quality standards"?

Reply:
🤷‍♂️ I don't see what might be ambiguous myself.
Making the below suggested change to I'll include this one too anyway.